### ISSUE TITLE
Bug - Th - PF warns about missing aria-label for elements that have aria-hidden="true"

### ISSUE THREAD

**Issue by mvollmer** [patternfly/patternfly#6658]
We have a empty `Th` and Patternfly rightfully demands it to be made accessible. However, in our case, the best thing is to hide the whole column from the screenreader with aria-hidden in every cell in that column. However, PF will still emit its warning for a `Th` component like this:

```
<Th aria-hidden="true" />
```

**Comment by mattnolting** [comment]
addressing in #6272

**Comment by mvollmer** [comment]
> addressing in #6272

But, that issue has label "wontfix"...

Also, I can see how you plan to fix #6643 in #6272, but this here is about changing the condition in Th.tsx for showing the warning to

```
!children && !screenReaderText && !ariaLabel && !ariaHidden == "yes"
```

Anyway, thanks a lot for addressing this!

**Comment by mattnolting** [comment]
> But, that issue has label "wontfix"...

"wontfix" has been removed

@thatblindgeye transferring this issue to pf react
